blacksheep/server/normalization.py

```diff
@@ -496,19 +518,7 @@ def get_async_wrapper_for_output(
 ) -> Callable[[Request], Awaitable[Response]]:
     @wraps(method)
     async def handler(request: Request) -> Response:
-        result = await method(request)
-
-        if result is None:
-            # later the application defaults to 204 No Content
-            return result
-
-        if not isinstance(result, Response):
-            # default to a plain text or JSON response
-            if isinstance(result, str):
-                return responses.text(result)
-            return responses.json(result)
-
-        return result
+        return ensure_response(await method(request))
```

Comment:
Maybe we can move these lines to variables? for better readability.

e.g

```python
result = await method(request)
response = ensure_response(result)
return response
```

Reply:
There is not consensus on this detail: some people, like me, prefer to avoid the declaration of variables that are used only once to reduce verbosity, others prefer more declarations for increased readability.